src/components/ActionBar.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="vxp-action-bar">
+    <span v-if="title && showTitle" class="vxp-action-bar__title">{{ title }}</span>
+    <slot></slot>
+  </div>
+</template>
+
+<script>
+import Gestures from '../mixins/GestureMixin';
+
+export default {
+  name: 'ActionBar',
+  data: function() {
+    return {
+      navigationButtonCreated: false,
+    };
+  },
+  props: {
+    title: String,
+  },
+  computed: {
+    showTitle: function() {
+      if (this.$slots.default) {
+        for (let slot of this.$slots.default) {
+          if (slot.componentOptions && slot.componentOptions.tag && !['ActionItem', 'NavigationButton'].includes(slot.componentOptions.tag)) return false;
+        }
+      }
+      return true;
+    },
+  },
+  mounted() {
+    // Only last navigation button will show
```

Comment:
since there are slots on this component, they may be modified on runtime. so it may be a good idea to process slots on 'updated' hook.

Reply:
Fixed and updated